canvasses.md

```diff
@@ -0,0 +1,470 @@
+    "created_date": "2014-03-20T21:04:31Z",
+    "modified_date": "2014-03-20T21:04:31Z",
+    "action_date": "2014-03-18T11:02:15Z",
+    "contact_type": "in-person",
+    "input_type": "mobile",
+    "failure_code": "",
+    "_links": {
+        "self": {
+            "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/canvasses/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3"
+        },
+        "osdi:target": {
+            "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29"
+        },
+        "osdi:answers": [
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/questions/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0111/answers/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0ccc"
+          },
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/questions/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0222/answers/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bbb"
+          }
+        ],
+        "osdi:taggings": [
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/tags/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0aaa/taggings/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0ddd"
+          },
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/tags/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bbb/taggings/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0eee"
+          }
+        ]
+    }
+}
+```
+
+_[Back to top...](#)_
+
+
+### Scenario: Creating a new Canvass (POST)
+
+Posting to the Canvasses collection endpoint and including a link to existing Answer and Tagging resource(s) will allow you to create a new canvass that indicates that the indicated Person was canvassed and that the answers and taggings indicated were recorded as a result.  The response is the new canvass that was created.
+
+#### Request
+
+```javascript
+POST https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/canvasses/
+
+Header:
+OSDI-API-Token:[your api key here]
+
+{
+    "identifiers": [
+        "foreign_system:1"
+    ],
+    "origin_system": "OpenSupporter",
+    "action_date": "2014-03-18T11:02:15Z",
+    "contact_type": "in-person",
+    "input_type": "mobile",
+    "failure_code": "",
+    "_links" : {
+        "osdi:answers": [
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/questions/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0111/answers/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0ccc"
+          },
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/questions/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0222/answers/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bbb"
+          }
+        ],
+        "osdi:taggings": [
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/tags/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0aaa/taggings/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0ddd"
+          },
+          {
+          	"href": "https://osdi-sample-system.org/api/v1/tags/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bbb/taggings/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0eee"
+          }
+        ]
```

Comment:
you're right, actually -- you can link to a taggings collecting on canvass, it would still be people that are tagged. It's like the submissions endpoint for forms (http://opensupporter.github.io/osdi-docs/submissions.html#links) which links to answers even though those answers themselves only relate to questions and people and not back to the submission.

So, instead of arrays, endpoints, but it's correct other than that I think.